install/installer/pkg/components/server/deployment.go

```diff
@@ -53,7 +54,6 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 			Name:      "gitpod-license-key",
 			MountPath: licenseFilePath,
 			SubPath:   "license",
-			ReadOnly:  true,
```

Comment:
Why is this necessary? :thinking: Maybe it make sense to make that explicit here, and add a comment that states why that is required?

Reply:
Arguably not, but it feels prudent to make readonly values explicitly readonly to me